src/app/course/_forms/goal.form.ts

```diff
@@ -32,6 +32,15 @@ export class GoalForm {
         })
     }
 
+    static createGoalItemFormInput(inCategory: number, inDifficulty: string, inNum: number): FormGroup {
+        const builder = new FormBuilder()
+        return builder.group({
+            category: new FormControl(inCategory, [Validators.required]),
+            difficulty: new FormControl(inDifficulty, [Validators.required]),
+            number_of_questions: new FormControl(inNum, [Validators.required])
```

Comment:
Let's rename this to numberOfQuestions. The convention is all the namings should be camelCase. But in some cases, because our API is in snake_case we had to compromise.
If I did this in my code too, I made a mistake.